README.rst

```diff
@@ -89,9 +89,12 @@ Follow the setup instructions of `openwisp-controller
     ]
 
     # Make sure you change them in production
-    INFLUXDB_USER = 'openwisp'
-    INFLUXDB_PASSWORD = 'openwisp'
-    INFLUXDB_DATABASE = 'openwisp2'
+    TIMESERIES_DB = {
+        'DATABASE': 'influxdb',
```

Comment:
I think this should be a path to a class, which will have to be imported dinamically.
That way we allow contributors to write their customized backends more easily.
(please document how to do this, with a short section that mentions this).

I also think the dictionary key should be named `BACKEND`